apps/financial-aid/web-veita/src/routes/PrintableApplication/printableApplication.tsx

```diff
@@ -47,9 +48,9 @@ const PrintableApplication = () => {
             isPrint={true}
           />
           {application?.files && (
-            <>
+            <Box marginTop={20}>
```

Comment:
We could add the margin when print under css print, should we maybe do that instead?

Reply:
I decided to have it here to create a bit of a margin between the timeline and the attachments. 
So before it looked like this:
![image](https://user-images.githubusercontent.com/43727721/162011256-bdc3db12-14dd-488d-94f3-c5d68ff520aa.png)

Now its like this: 
![image](https://user-images.githubusercontent.com/43727721/162011323-c3fb5431-c567-48d8-8772-908c5bddf295.png)